lib/logger/lib/logger/watcher.ex

```diff
@@ -89,13 +93,13 @@ defmodule Logger.Watcher do
   end
 
   def handle_info({:gen_event_EXIT, handler, reason}, {mod, handler, _} = state) do
-    _ = Logger.error "GenEvent handler #{inspect handler} installed at #{inspect mod}\n" <>
+    _ = Logger.error ":gen_event handler #{inspect handler} installed at #{inspect mod}\n" <>
                  "** (exit) #{format_exit(reason)}"
     {:stop, reason, state}
   end
 
   def handle_info({:DOWN, ref, _, _, reason}, {mod, handler, ref} = state) do
-    _ = Logger.error "GenEvent handler #{inspect handler} installed at #{inspect mod}\n" <>
+    _ = Logger.error ":gen_event handler #{inspect handler} installed at #{inspect mod}\n" <>
```

Comment:
We can never reach here because `:gen_event` will link to this process, so we can skip the monitoring altogether. Either we will get `:gen_event_EXIT` or an abnormal exit signal and exit abruptly before being able to handle a `:DOWN`.